stdlib/public/core/UnicodeScalar.swift

```diff
@@ -34,6 +34,7 @@ public struct UnicodeScalar :
   }
 
   /// Creates an instance of the NUL scalar value.
+  @swift3_migration(message="use 'UnicodeScalar(\"\\0\")'")
```

Comment:
`UnicodeScalar("\0")!`, but `"\0" as UnicodeScalar` would be better since it does not require force-unwraps.